src/commons/utils/cluster_utils.py

```diff
@@ -139,28 +140,21 @@ def collect_support_bundles(self, dir_path: str) -> tuple:
         LOGGER.info("Support bundle '%s' generated and copied to %s.", file_name, local_path)
         return os.path.exists(local_path), local_path
 
-    def send_k8s_cmd(
-            self,
-            operation: str,
-            pod: str,
-            namespace: str,
-            command_suffix: str,
-            decode=False,
-            **kwargs) -> bytes:
-        """send/execute command on logical node/pods."""
+    def send_k8s_cmd(self, operation: str, pod: str, namespace: str, command_suffix: str,
```

Comment:
decode option is not needed?

Reply:
This change required to fix codacy, now we are using it from kwargs.